src/app/modules/item/components/chapter-children/chapter-children.component.html

```diff
@@ -3,8 +3,39 @@
 <p *ngIf="state === 'error'">Error while loading the group managers</p>
 
 <ng-container *ngIf="state === 'ready'">
-  <p class="validation-text">
+  <p class="validation-text" *ngIf="case === 'normal'">
     To validate this chapter, solve at least all tasks with Validation type
   </p>
-  Ready, loaded {{children.length}} children.
+  <p class="validation-text" *ngIf="case === 'empty'">
+    This chapter has no content visible to you, so you can't validate it for now.
+  </p>
+
+  <ul class="data-list-container activity">
+    <li *ngFor="let item of children; index as ln">
+      <span class="activity-progress">
+        <alg-score-ring
+          *ngIf="!item.isLocked && item.result"
+          [diameter]="30"
+          [currentScore]="item.result.current_score"
+          [isValidated]="item.result.is_validated"
+          [bestScore]="item.best_score"
+        ></alg-score-ring>
```

Comment:
the score should be displayed inside the ring

Reply:
fixed